content/tbb/why-isp-not-block-tor/contents.lr

```diff
@@ -0,0 +1,19 @@
+_model: question
+---
+title: Why doesn't the ISP just block Tor?
+---
+seo_slug: why-isp-not-block-tor
+---
+description:
+### Does my ISP know if I'm using Tor?
+Your Internet Service Provider (ISP), may be able to see that you're using Tor, but they won't know where you're going when you do.
+You can prevent your ISP from knowing this by using a VPN. 
+Using a VPN with Tor is not recommended, so you should consider using [bridges](https://tb-manual.torproject.org/bridges/) instead. 
+
+### Can Tor be blocked?
+Yes, some governments or network admistrators may be able to block Tor. They can get the list of Tor relays and block them.
+
+### If a company bans you from using Tor, what can you do to continue using?
+You can bypass this by using [bridges](https://tb-manual.torproject.org/bridges/). 
```

Comment:
I'm merging Circumvention Tools and Pluggable Transports page. 
The new one will be /circumvention.